README.md

```diff
@@ -2,6 +2,38 @@
 
 This is the userspace API for the robot, which is what students will interface with to program their robots.
 
+# Installation instructions
+
+## Building the debian package
+
+``` bash
+# Install build tools:
+sudo apt install build-essential devscripts debhelper equivs
+
+# cd to the root of this project
+cd path/to/robotd
+
+# Install dependencies
+sudo mk-build-deps -ir
+
+# Build the package:
+debuild -uc -us
+
+```
+
+## Instaling the debian package
```

Comment:
"Installing"

Reply:
Huh, the commit message was also wrong, it appears I don't know how to spell intsall.